src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -126,6 +128,8 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
 
   // =========================TO REMOVE=======================
 
+  const firstbootFlag = useFlag('image-builder.firstboot.enabled');
+  const isFirstBootEnabled = isBeta() && firstbootFlag;
```

Comment:
Given we should use it to hide the review as well, should this live in Utilities?
Something like https://github.com/osbuild/image-builder-frontend/blob/main/src/Utilities/useExperimentalFlag.ts

Reply:
What if we had a single hook, `useFirstBootEnabled`, that would return true or false depending on whether we were in stage or prod _and_ whether the flags `image-builder.firstboot.stable.enabled` or image-`builder.firstboot.beta.enabled` were on/off?